Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -0,0 +1,464 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+require_relative "../livecheck_strategy"
```

Comment:
Could this be:

```suggestion
require "lvecheck_strategy"
```

?

Reply:
Yeah that works, making the change 👍🏼